docs/source/getting-started/first-feature.rst

```diff
@@ -19,7 +19,12 @@ This means:
 
 We will see that DAML lets us implement these guarantees in a direct and intuitive way.
 
-There are two parts to building the messaging feature: the DAML model and the UI.
+There are three parts to building and running the messaging feature: 
+
+    1. Adding the necessary changes to the DAML model  
+    2. Making the corresponding changes in the UI
+    3. Running the new feature. In order to do that we need to terminate the previous ``./daml-start.ch`` process and run it again.  
```

Comment:
There's a typo. It should be `./daml-start.sh` (with `sh` in the end, not `ch`.)

Reply:
```suggestion
    3. Running the new feature. In order to do that we need to terminate the previous ``./daml-start.sh`` process and run it again.  
```